types/semantic-ui-transition/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for semantic-ui-transition 2.2
+// Project: http://www.semantic-ui.com
+// Definitions by: Leonard Thieu <https://github.com/leonard-thieu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="jquery" />
+
```

Comment:
Sorry, could you clarify? Are you saying I need 4 slashes total on the version line?